### DISCUSSION TITLE
Error from Makefile and code deployment

### DISCUSSION THREAD

**Discussion by GiftedView** [contiki-ng/contiki-ng#2410] [General]
I am deploying some codes on the Cooja Simulator and i am getting this error. How do i resolve it.
![error](https://user-images.githubusercontent.com/52604514/224454326-848c71c6-dbcb-4425-a969-874e2c11ad37.jpg)

**Comment by pjonsson** [comment]
My guess is that something went wrong when detecting the Cooja or Contiki-NG directory.

I don't have a Cooja available right now but in one of the right-most menus there is an option that brings up a dialog box with a lot of paths, check that those paths are correct for your system. You can also get a bit more information from make with `make main.sky TARGET=sky Q=''`.
